LEGO1/omni/include/mxpresenterlist.h

```diff
@@ -23,18 +23,24 @@ class MxPresenterList : public MxPtrList<MxPresenter> {
 	// MxPresenterList::`scalar deleting destructor'
 };
 
+DECOMP_SIZE_ASSERT(MxPresenterList, 0x18)
```

Comment:
size asserts should never be in a header - add it to any one of the CUs that uses a list

Reply:
my bad, these were already in a cpp file. I have removed the redundant new ones.